refresh/index.js

```diff
@@ -838,10 +842,13 @@ module.exports = Generator.extend({
       this.dependencies.push('.Package(url: "https://github.com/IBM-Swift/Health.git", majorVersion: 0),')
     }
 
-    var initCodeForEndpoints = endpointNames.map(name => `initialize${name}Routes()`)
+    var initCodeForEndpoints = endpointNames.map(name => `initialize${name}Routes(app: self)`)
     this.appInitCode.endpoints = this.appInitCode.endpoints.concat(initCodeForEndpoints)
 
-    if (this.hostSwagger) this.appInitCode.endpoints.push(`initializeSwaggerRoutes(path: projectPath + "/definitions/${this.projectName}.yaml")`)
+    if (this.hostSwagger) {
+      this.appInitCode.endpoints.push(`initializeSwaggerRoutes(app: self)`)
+      this.appInitCode.swaggerPath = `let swaggerPath = projectPath + "/definitions/${this.projectName}.yaml"`
```

Comment:
Feels like the `swaggerPath` doesn't really belong in `appInitCode`, even with its new wider usage. Perhaps a separate property, or just something directly inside the template?

Reply:
put into a separate property